CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Internal server error for git sub modules without tree object ([#1397](https://github.com/scm-manager/scm-manager/pull/1397))
 - Do not expose subversion commit with id 0 ([#1395](https://github.com/scm-manager/scm-manager/pull/1395))
+- Repositories can now be cloned with api keys ([#1407](https://github.com/scm-manager/scm-manager/pull/1407))
```

Comment:
I'm not sure if this is correct. I thought only mercurial repositories and clones over ssh with api keys are affected.

Reply:
We tested it and it does work over ssh with an api key. So it is only a problem with Mercurial.